### PR TITLE
Mixer calibrations update

### PR DESCRIPTION
**Pull request by AdriaanRol**
Updated mixer calibrations from @NielsBultink 's branch. 

Included a bugfix for mixer skewness calibrations with CBoxv3 (debugged with @gtaifu ). 

Now we have beautiful numerical mixer calibrations as standard. 
image
image

### CODE REVIEW

File: pycqed/measurement/calibration_toolbox.py [hunk 1]
```diff
@@ -583,34 +598,32 @@ def mixer_skewness_cal_UHFQC_adaptive(UHFQC, SH, source, AWG,
...
+        Navg=5, delay=0.0, prepare_each_point=False)
 
     ad_func_pars = {'adaptive_function': nelder_mead,
-                        'x0': [1.0, 0.0],
-                        'initial_step': [.15, 10],
-                        'no_improv_break': 10,
-                        'minimize': True,
-                        'maxiter': 500}
+                    'x0': [1.0, 0.0],
+                    'initial_step': [.15, 10],
```
Comment (NielsBultink) [inline pycqed/measurement/calibration_toolbox.py]: I would change the no_improvement break to 15 as I have found 10 to be insufficient for reliable convergence.

File: pycqed/measurement/calibration_toolbox.py [hunk 2]
```diff
@@ -34,41 +34,52 @@ def mixer_carrier_cancellation_duplexer(**kw):
...
-
-    chI_par(ch_min[0])
-    chQ_par(ch_min[1])
-    return ch_min
+    '''
+    Make coarse sweeps to approximate the minimum
+    '''
+
+    detector = det.Signal_Hound_fixed_frequency(
+        SH, frequency=(source.frequency()),
```
Comment (NielsBultink) [inline pycqed/measurement/calibration_toolbox.py]: please set the reference level to -40 dB to improve noise floor of the signal hound with:

SH.ref_level(-40)

**Comment by elrama-** [comment]
seems reasonable. I would say we just need to test it. 

Are those plots from a real run of the script? (then it is tested)

**Comment by AdriaanRol** [comment]
@NielsBultink I have added the SH_ref_level as you suggested. 
@elrama- this is real data.

**Comment by AdriaanRol** [comment]
@elrama- , @NielsBultink  Could either one of you approve this PR?

**Pull request merged**